cves/kernel/CVE-2022-27223.yml

```diff
@@ -278,8 +271,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: This vulnerability involved indexing an array out of bounds via USB connection, when the USB device was handling a status check or updating a feature.
```

Comment:
I don't think this feature had any inter-process communication, if so then expand a little bit more on how inter-process communication is used within this affected feature.